src/interfaces/banner.ts

```diff
@@ -94,6 +94,10 @@ export type Action =
       label: 'Check'
       actionName: 'confirm-temp-seed'
     }
+  | {
+      label: 'Open'
+      actionName: 'open-confetti-modal'
```

Comment:
Here it again sounds too generic as well. What if in 1 month we have another confetti modal?